src/Windows/Host/Client/Impl/Host/LocalClientServices.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.R.Host.Client {
+    public class LocalClientServices : ILocalClientServices {
```

Comment:
agreed and removed.

---
In reply to: [111822088](https://github.com/Microsoft/RTVS/pull/3419#discussion_r111822088) [](ancestors = 111822088)